### PR TITLE
LPS-176131 | LPS-176132 Integration tests for 404 OOTB

### PR DESCRIPTION
**Pull request by jkappler**
## Motivation

[Link to ticket](https://issues.liferay.com/browse/LPS-176131)
[Link to ticket](https://issues.liferay.com/browse/LPS-176132)

Adds integration tests for blank site initializer and action command

## Test Scenario

* Execute the integration tests in the PR

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by ruben-pulido** [comment]
ci:test:relevant

**Comment by ruben-pulido** [comment]
ci:test:sf

File: modules/apps/layout/layout-admin-web-test/src/testIntegration/java/com/liferay/layout/admin/web/internal/portlet/action/test/AddLayoutUtilityPageMVCActionCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+		ServiceContextThreadLocal.pushServiceContext(_serviceContext);
+	}
+
+	@After
+	public void tearDown() {
+		ServiceContextThreadLocal.popServiceContext();
+	}
+
+	@Test
+	public void testAddLayoutUtilityPageFromTemplate() throws Exception {
```
Comment (ruben-pulido) [inline modules/apps/layout/layout-admin-web-test/src/testIntegration/java/com/liferay/layout/admin/web/internal/portlet/action/test/AddLayoutUtilityPageMVCActionCommandTest.java:104]: Should we rename the method to testAddLayoutUtilityPageEntry?

We are not creating the Utility Page based on a Page Template as far as I can see.

**Comment by ruben-pulido** [comment]
https://github.com/liferay-echo/liferay-portal/pull/11201

**Pull request closed without merging**